core/modules/widgets/navigator.js

```diff
@@ -549,13 +549,25 @@ NavigatorWidget.prototype.handlePerformImportEvent = function(event) {
 	var self = this,
 		importTiddler = this.wiki.getTiddler(event.param),
 		importData = this.wiki.getTiddlerDataCached(event.param,{tiddlers: {}}),
-		importReport = [];
+		importReport = [],
+	    	moreTags = this.wiki.getTiddlerText("$:/language/DefaultNewTiddlerTags");
 	// Add the tiddlers to the store
 	importReport.push($tw.language.getString("Import/Imported/Hint") + "\n");
 	$tw.utils.each(importData.tiddlers,function(tiddlerFields) {
 		var title = tiddlerFields.title;
 		if(title && importTiddler && importTiddler.fields["selection-" + title] !== "unchecked") {
-			var tiddler = new $tw.Tiddler(tiddlerFields);
+			var tiddler;
+			if(moreTags && tiddlerFields.tags) {
+				tiddler = new $tw.Tiddler(tiddlerFields,{
+					tags: tiddlerFields.tags + " " + moreTags
+				});
+			} else if (moreTags) {
+				tiddler = new $tw.Tiddler(tiddlerFields,{
+					tags: moreTags
+				});
+			} else {
+				tiddler = new $tw.Tiddler(tiddlerFields);
+			}
```

Comment:
I do have strong concerns about these changes!  IMO the import mechanism should _NOT_ modify content in an invisible and "aggressive" way.

Reply:
For me this is an absolute no-go. ... IFF there is a possibility to automatically add tags, it has to be an OPT-IN and switched off by default. 

I also don't see a reason, why an option to add tags with "new - something" buttons needs to add tags for hundreds of imported tiddlers. That' doesn't make sense, since it makes tags useless.

Reply:
@pmario - that are good points! I'll revert the changes for the import mechanism